### PR TITLE
move run above and below

### LINKED ISSUE #6910: Run line by line: movement of button is jarring
Testing microsoft/vscode#129467

I was expecting the run line by line button to stay in the same location the entire time so that I could quickly click through the cell without needing to move my cursor. Needing to move my cursor depending upon the timing/point in the journey felt unexpected. 

recording (38)

Does the button have to move all the way to the left once it starts running?

### PR DESCRIPTION
**Pull request by DavidKutu**
to position 1 instead of 0, this allows run by line to be the first menu item

This PR fixes https://github.com/microsoft/vscode-jupyter/issues/6910

### CODE REVIEW

**Review by roblourens — CHANGES_REQUESTED** [review]
Use the `CellToolbarOrder` enum for this. I guess I didn't adopt it for the run above/below commands, you could do that too if you want

File: src/vs/workbench/contrib/notebook/browser/contrib/coreActions.ts [hunk 1]
```diff
@@ -420,7 +420,8 @@ registerAction2(class ExecuteAboveCells extends NotebookMultiCellAction<INoteboo
 				},
 				{
 					id: MenuId.NotebookCellTitle,
-					group: 'inline',
+					order: CellToolbarOrder.SplitCell,
```
Comment (roblourens) [inline src/vs/workbench/contrib/notebook/browser/contrib/coreActions.ts:423]: I mean, add new entries that represent this command :)

**Review by roblourens — APPROVED** [review]

**Comment by scybert** [comment]
Hi @roblourens . Is merging of failed PR's expected in VScode? I'm totally new!

**Comment by roblourens** [comment]
I checked that the errors were not related to these changes

**Pull request merged**